components/textfield/index.css

```diff
@@ -1265,7 +1265,9 @@ governing permissions and limitations under the License.
 .spectrum-Textfield--multiline {
   --spectrum-textfield-input-line-height: normal;
 
-  .spectrum-Textfield-input {
+	.spectrum-Textfield-input {
+		resize: both;
+		block-size: unset;
```

Comment:
👍 I'm glad you got this update in there. In addition to the XL height bug fix you mentioned in the PR description, it also lets the textarea height size appropriately if the rows attribute it used, e.g. `rows="5"`.

Reply:
While I still think that using `block-size: unset;` is a good idea, I've removed it from this PR as it causes a few issues downstream with SWC and with the quiet variants. 

Text areas default to two rows in most browsers, resulting in fields like quiet that previously displayed as one line, displaying as two by default. It also would change the heights on some fields. This is significant enough of a change to be its own PR and can be revisited later, likely as part of the Spectrum 2 migration. It would likely require work on the SWC end.

The benefits of keeping the change would be the ability to use the `rows` attribute on the textarea. SWC lists rows in their API for Text area but it does not seem to work currently. The design specs do not specify a fixed height, only  a minimum height.